### PR TITLE
Deploy regional demo in release script

### PR DESCRIPTION
**Pull request by ltrung**
**Description of changes:**
- Deploy a meeting demo with chime SDK meetings in release script.
- Fix some minor bugs in version bump script (CHANGELOG date in UTC)

**Testing:**
Manually deploy the regional meeting demo.

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
N/A

**Checklist:**

1. Have you successfully run `npm run build:release` locally? N/A

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? No

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: script/version-utils.js [hunk 1]
```diff
@@ -54,14 +54,26 @@ const getNewVersion = (currentVersion, versionIncrement) => {
...
+  // Return local date in ISO format yyyy-mm-dd
+  const today = new Date();
+  const year = today.getFullYear();
+
+  const month = new Intl.DateTimeFormat('en-US', {month: "2-digit"}).format(today);
+
+  const day = new Intl.DateTimeFormat('en-US', {day: "2-digit"}).format(today);
+
+  return year + "-" + month + "-" + day;
+}
```
Comment (xuesichao) [inline script/version-utils.js:67]: nit: could we remove these empty lines?

File: script/version-utils.js [hunk 2]
```diff
@@ -131,7 +143,7 @@ const versionBump = async (option, branchName) => {
   logger.log(`Do you want to upload these files to ${branchName} branch?\n`);
   await shouldContinuePrompt();
   spawnOrFail('git', [`push origin HEAD:${branchName} -f`]);
-  if (branchName === 'version-bump') {
+  if (branchName.startsWith('version-bump')) {
```
Comment (xuesichao) [inline script/version-utils.js:146]: Just for understanding, is there any case that the branch name will be `version-bump*`?

**Review by xuesichao — APPROVED** [review]

File: script/version-utils.js (hunk 2 above)
Comment (ltrung) [reply script/version-utils.js:146]: No it should be version-bump-version (used to be just version-bump but that can cause conflicts when we try to release in multiple version.

File: script/version-utils.js (hunk 1 above)
Comment (ltrung) [reply script/version-utils.js:67]: next time :p

**Pull request merged**